src/snowflake/snowpark/dataframe.py

```diff
@@ -1893,14 +1919,33 @@ def with_column(self, col_name: str, col: Column) -> "DataFrame":
             ------------------------
             <BLANKLINE>
 
+        Example 2::
+
+            >>> from snowflake.snowpark.functions import udtf
+            >>> @udtf(output_schema=["number"])
+            ... class sum_udtf:
+            ...     def process(self, a: int, b: int) -> Iterable[Tuple[int]]:
+            ...         yield (a + b, )
+            >>> df = session.create_dataframe([[1, 2], [3, 4]], schema=["a", "b"])
+            >>> df.with_column("total", sum_udtf(df.a, df.b)).show()
+            -----------------------
+            |"A"  |"B"  |"TOTAL"  |
+            -----------------------
+            |3    |4    |7        |
+            |1    |2    |3        |
+            -----------------------
+            <BLANKLINE>
```

Comment:
The row order in output for `show()` is not the same all the time causing doctests to fail. How should I structure this to pass them?

Reply:
Let's maybe add a `sort(col("A"))`? Maybe there are better ways that I'm not aware of.